### ISSUE TITLE
Bug since last release: admins can't access tribe settings

### ISSUE THREAD

**Issue by ralphboeije** [part-up/part-up#306]
To reproduce:
1. log in as regular user (no super admin) that is a tribe admin
2. Go to your tribe settings
3. see the not accessible notice appear
4. reload url and see the settings be accessible

Intended result: be able to view and edit your tribe settings on first attempt.

![tribe settings unavailable](https://cloud.githubusercontent.com/assets/14044235/13931960/204ecdf0-efa6-11e5-96db-04f854f10c7e.png)

**Comment by ralphboeije** [comment]
What I notice continuously is that first I get the 404 and then it seems to auto refresh and show me the content. But that might be because I am super admin?

**Comment by ralphboeije** [comment]
Maybe this is also related: i get a 404, but with placeholder copy. @tlimpanont I can still see this placeholder copy in z-ft-tribestart-sprint i18n files. Shouldn't they have been removed? But nonetheless it is strange that I can get this copy at all I think?

**Comment by peterpeerdeman** [comment]
I've deployed 1.17 (same version as production) on acceptance and tried to reproduce, but could not reproduce it consistently. Can you try to find a case in which it fails every time?

**Comment by ralphboeije** [comment]
A couple of things I can reproduce constantly is:
- Getting the 404 a few sub-seconds before loading the contents of a closed tribe (tried it both with admin account and my own account), e.g. https://pu-acceptance.lifely.nl/tribes/test-private-tribe?show=false
- When typing /admin to go to admin panel, you first see a 404 before it loads the admin panel

I can't reproduce the issue listed in the first comment though. However, I can see a reload taking place every time you visit a closed tribe. It seems to be happening right before the archived filter and start a part-up in this tribe link are loaded.

And there is the unrelated issue of getting a 404 with wrong copy when visiting a non existing tribe https://pu-acceptance.lifely.nl/tribes/ing

**Comment by ralphboeije** [comment]
@peterpeerdeman this bug might also be related to this issue: tribe settings unavailable message appears wrongly. To reproduce (consistently in acceptance):
- log in to acceptance
- assign admin rights to a regular user that is member of a tribe
- go to the tribe settings and see them appear
- click the uppers tab and see the "tribe settings unavailable" 404 appear (see screen below), bulk invite and tribe info do work though.

<img width="1440" alt="screen shot 2016-03-23 at 13 31 33" src="https://cloud.githubusercontent.com/assets/14044235/13985392/be86f720-f0fc-11e5-8821-3ed062e176ba.png">

**Comment by eriksoon** [comment]
Did you guys also notice the typo in the prt scrn? instellings niet toegankelijk

**Comment by andreleon** [comment]
The check to see if a tribe was closed for a user was missing a check for an admin user.

When the admin of a tribe is re-assigned to a normal user that is not a member of the tribe, the user is made into an admin, but not to a member of the tribe. 

The same call that makes the user an admin should also make the user a member of the tribe. I will leave this up to @prik but for now I did a quick fix that prevents the 404 page from being shown and allows admins in the settings menu, but the problem that caused the issue still hasn't been fixed.
